src/tests/ftest/launch.py

```diff
@@ -886,6 +895,13 @@ def run_tests(test_files, tag_filter, args):
                                          args):
                     return_code |= 256
 
+    test_logs_lnk = os.path.join(avocado_logs_dir, "latest")
+    try:
+        os.remove(test_logs_lnk)
+        print("Removing sym link latest")
+    except OSError as error:
+        print("Failed to remove sym link latest: {}".format(error))
```

Comment:
Should this not update `return_code` to indicate that something failed?

Reply:
I think we should make this a warning instead and not change the return code.

Reply:
So when it happens, how does anyone know?  People don't look at the logs for every run they do looking for _warnings_.

Reply:
Retained this code in the latest commit. I think it should not change the return code here. Worst case, the sym link is not removed and shows up in the archive but we will still get results.